docs/semgrep-supply-chain/glossary.md

```diff
@@ -68,7 +68,7 @@ See [Overview of Semgrep Supply Chain](/semgrep-supply-chain/overview) to learn
 
 A type of Semgrep Supply Chain rule that performs reachability analysis. A reachability rule can determine if the vulnerable piece of code from a dependency is used in the codebase that imports it.
 
-Compare its opposite: [Lockfile-only rules](#lockfile-only-rules).
+Compare its opposite: [rules without reachability analysis](#rules-without-reachability-analysis)
```

Comment:
If you accept the above comment about **No-reachability rules** then update this header as well.